mayastor/src/bin/cli/nexus_child_cli.rs

```diff
@@ -0,0 +1,64 @@
+//!
+//! methods to interact with the nexus child
+
+use crate::context::Context;
+use ::rpc::mayastor as rpc;
+use clap::{App, AppSettings, Arg, ArgMatches, SubCommand};
+use tonic::Status;
+
+pub async fn handler(
+    ctx: Context,
+    matches: &ArgMatches<'_>,
+) -> Result<(), Status> {
+    match matches.subcommand() {
+        ("fault", Some(args)) => fault(ctx, &args).await,
+        (cmd, _) => {
+            Err(Status::not_found(format!("command {} does not exist", cmd)))
+        }
+    }
+}
+
+// TODO: Make subcommands use the name of the child rather than the uri
```

Comment:
I don't think we can do this because all replicas in k8s have the same name?

Reply:
This was a copy-paste error. I've removed this comment together with where I copied it from given that we can't do this.